.github/workflows/ci.yml

```diff
@@ -1,5 +1,16 @@
 name: CI
-on: [push]
+on:
+  # We would like to trigger for CI for any pull request action -
+  # both from QuantCo's branches as well as forks.
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
The default types are usually sufficient (this also misses `reopened` ;). Since none of the jobs below seem to be dependent on labels, I don't think it's necessary to use a custom list here.

Reply:
Like so?
https://github.com/Quantco/metalearners/pull/74/commits/13d61eebb7e52af1270c111456e777e5805ed882